mobile/src/bundles/common/components/modal/modal.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+
+import {
+    Button,
+    RNModal,
+    Text,
+    TouchableOpacity,
+    View,
+} from '~/bundles/common/components/components';
+import { ButtonType, TextCategory } from '~/bundles/common/enums/enums';
+import { globalStyles } from '~/bundles/common/styles/styles';
+
+import { styles } from './styles';
+
+type Properties = {
+    title: string;
+    visible: boolean;
+    acceptText?: string;
+    children: React.ReactNode;
+    onClose: () => void;
+    onAccept?: () => void;
+};
+
+const Modal: React.FC<Properties> = ({
```

Comment:
And this one should be called "ActionModal" (it requires some action from user, because it has buttons)